content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md

```diff
@@ -0,0 +1,157 @@
+---
+title: delfin
+description: "delfin deployment on kubernetes guide."
+tags: ["installation guide", "delfin"] 
+---
+
+## Delfin deployment on kubernetes cluster
+Here are the steps to deploy delfin on a kubernetes  cluster 
+
+### Pre-config (Ubuntu 18.04)
+The installation is tested on Ubuntu 18.04, please make sure to use the right one.
+Root user is suggested for the installation.
+
+* packages
+
+Install following packages:
+```bash
+apt-get install -y libltdl7 libseccomp2 git curl wget make
+```
+
+* docker
+Install docker:
+```bash
+wget https://download.docker.com/linux/ubuntu/dists/xenial/pool/stable/amd64/docker-ce_18.03.1~ce-0~ubuntu_amd64.deb
+dpkg -i docker-ce_18.03.1~ce-0~ubuntu_amd64.deb 
+```
+
+* Kind
+Install kind:
+```bash
+curl -Lo ./kind https://github.com/kubernetes-sigs/kind/releases/download/v0.11.1/kind-linux-amd64
+chmod +x ./kind
+mv ./kind /some-dir-in-your-PATH/kind
+```
+Create a kubernetes cluster:
+```bash
+Kind create cluster
+```
+Check created cluster:
+Example:
+```bash
+root@proxy:~# kind get clusters
+kind
+```
+* kubectl
+ Install kubectl:
+ 
+ Download the latest release with the command:
+```bash
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+```
+ Note:
+ 
+If you do not have root access on the target system, you can still install kubectl to the ~/.local/bin directory:
+```bash
+chmod +x kubectl
+mkdir -p ~/.local/bin/kubectl
+mv ./kubectl ~/.local/bin/kubectl
+# and then add ~/.local/bin/kubectl to $PATH
+```
+Test to ensure the version you installed is up-to-date:
+```bash
+kubectl version --client
+```
+### Build the delfin docker image locally
+Download the source code of delfin:
+```bash
+git clone https://github.com/sodafoundation/delfin.git
+```
+
+Build Delfin Image with k8s tag locally: 
```

Comment:
Where is the process to build the docker image? Where can the tag be changed? If this documentation is available separately provide the link, else please mention the steps here.

Reply:
provided